### DISCUSSION TITLE
.Net Framework Version Install for Clients

### DISCUSSION THREAD

**Discussion by PhillipBrickner** [immense/Remotely#264] [Q&A] [answered]
Hi,
I have been using Remotely for a while and went ahead and installed the latest version and I think I have the wrong .Net installed.

I now have Remotely Version: 2021.03.17.0718 installed and I am getting alerts from the clients that say "The .NET Framework version on this device is outdated, and Remotely will no longer receive updates. Update the installed .NET Framework version to fix this."
Everything is still working I am just getting that error.

I installed what I thought was the correct ones the file was the dotnet-sdk-5.0.201-win-x64.exe which after installing allowed for the client to update. 

I believe this is why when I try to go to the latest version of Remotely it no longer allows for connections.

Any help on this from anyone would be appreciated.  This is not an issue just user error.

**Comment by bitbound** [comment, marked as answer]
I thought I answered this.  I'm sorry!

This message is referring to .NET Framework 4.8.  I updated the Windows installer to this version (latest and final version), but I realized computers that were missing it wouldn't be able to auto-upgrade.  So I put this message in here.

You can find the installer for it at the below link.  You want the `Runtime`, not the `Dev Pack`.

(Side note.  In the future, I plan to make a single, cross-platform installer that's completely self-contained, so there won't be any external dependencies.)

https://dotnet.microsoft.com/download

![image](https://user-images.githubusercontent.com/20995508/114408361-2f6f8180-9b5e-11eb-9a3a-430702010ea7.png)

**Comment by PhillipBrickner** [comment]
Thanks Jared!  Super excited about a self contained installer! 
I don't know how you manage doing all this.   You must be able to write code in your sleep!
